### PR TITLE
This issue refers to issue #9, to create student medical history schema.

### PR DESCRIPTION
**Pull request by Shivam-53**

### CODE REVIEW

File: models/std_med_history.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Schema } from 'mongoose';
```
Comment (Hitansh159) [inline models/std_med_history.js:1]: no need for this import

File: models/std_med_history.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { connector } from './databaseUtil';
+
+const studentMedicalSchema= new Schema({
```
Comment (Hitansh159) [inline models/std_med_history.js:4]: studentMedicalSchema will be just a object no need for `new Schema`

File: models/databaseUtil.js
Comment (Hitansh159) [inline models/databaseUtil.js:1]: restore it to its previous state

**Review by Hitansh159 — CHANGES_REQUESTED** [review]
just small changes

File: models/databaseUtil.js
Comment (Shivam-53) [reply models/databaseUtil.js:1]: Done

File: models/std_med_history.js (hunk 1 above)
Comment (Shivam-53) [reply models/std_med_history.js]: Done

File: models/std_med_history.js (hunk 2 above)
Comment (Shivam-53) [reply models/std_med_history.js]: Done

File: models/std_med_history.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { connector } from './databaseUtil';
+
+const studentMedicalSchema=({
+    bloodGroup:{type:String, required:true},
+    pastMedicalHistory:{type:String, required:true},
+    immunisationHistory:{type:String, required:true},
+    chronicMedicalConditions:{type:String, required:true},
```
Comment (TejasNair9977) [inline models/std_med_history.js:7]: this is not required

File: models/std_med_history.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    parentsEmailId:{type:String, required:true},
+    parentsContact:{type:String, required:true},
+    relativeContacts:{type:String, required:true},
```
Comment (TejasNair9977) [inline models/std_med_history.js:10]: can you make these contact fields as numbers

File: models/std_med_history.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { connector } from './databaseUtil';
+
+const studentMedicalSchema=({
```
Comment (TejasNair9977) [inline models/std_med_history.js:3]: remove the extra brackets

**Review by TejasNair9977 — CHANGES_REQUESTED** [review]
other than these changes, it looks good to me

File: models/std_med_history.js (hunk 3 above)
Comment (Shivam-53) [reply models/std_med_history.js]: Done

**Review by TejasNair9977 — APPROVED** [review]
looking good!

**Review by Hitansh159 — APPROVED** [review]

**Pull request merged**